plutus-tx/src/PlutusTx/Builtins.hs

```diff
@@ -611,6 +613,10 @@ bls12_381_G1_scalarMul :: Integer -> BuiltinBLS12_381_G1_Element -> BuiltinBLS12
 bls12_381_G1_scalarMul = BI.bls12_381_G1_scalarMul
 {-# INLINEABLE bls12_381_G1_scalarMul #-}
 
+bls12_381_G1_multiScalarMul :: BI.BuiltinList Integer -> BI.BuiltinList BuiltinBLS12_381_G1_Element -> BuiltinBLS12_381_G1_Element
```

Comment:
```suggestion
bls12_381_G1_multiScalarMul :: [Integer] -> [BuiltinBLS12_381_G1_Element] -> BuiltinBLS12_381_G1_Element
```
@kwxm, should this not be real lists? As this module is a user-oriented lib, the same applies for the G2 definition. I went for the current approach as the other builtins also have as `BI.BuiltinList` wrapper.